README.md

```diff
@@ -9,8 +9,11 @@ Using your AWS API credentials, S3Proxy allows you to make unauthenticated HTTP
 
 This makes it easy for tools like curl to access private buckets:
 
+Eg. Assuming you have a S3 bucket named `bucket1` and it has the following 
+structure `folder1/folder2/item.txt`. 
+
 ```bash
-curl http://localhost:9090/object/key
+curl http://localhost:9090/bucket1/folder1/folder2/item.txt?aws_region=eu-west-1
```

Comment:
Buckets are suppose to be global resources and by hitting the global s3 endpoint you should be able to get to the bucket. However, I could not get this to work without initiating the client with region. I think the SigV4 signing requires a location header? To be honest I'm not sure and will have to dig further into it.